platform.mk

```diff
@@ -90,10 +90,14 @@ TARGET_KEYMASTER_V4 := true
 # DSP
 TARGET_NEEDS_AUDIOPD := true
 TARGET_NEEDS_ADSP_SENSORS_PDR := true
+TARGET_HAS_CDSP := true
 
 # VPP
 TARGET_DISABLE_QTI_VPP := true
 
+# Hvdcp_opti
```

Comment:
Feel free to use this comment to explain what this does/is for, instead of repeating the name of the property. Otherwise you might as well omit it.